store/postgres/src/relational.rs

```diff
@@ -257,6 +264,17 @@ impl Layout {
     pub fn as_ddl(&self) -> Result<String, fmt::Error> {
         let mut out = String::new();
 
+        // Output enums first
+        for (name, values) in &self.enums {
+            let mut sep = "";
+            let name = SqlName::from(name.as_str());
+            write!(out, "create type {}.{}\n    as enum (", self.schema, name)?;
+            for value in values {
+                write!(out, "{}'{}'", sep, value)?;
+                sep = ", "
```

Comment:
Could you use [join](https://doc.rust-lang.org/std/slice/trait.SliceConcatExt.html#tymethod.join) here?

Reply:
I was trying to avoid allocating a `Vec` just so we can join it